rmgweb/database/forms.py

```diff
@@ -228,21 +228,20 @@ def clean_temp(self):
         """
         Custom validation for the temperature field to ensure that a valid temperature has been provided
         """
+        temp = self.cleaned_data['temp']
         try:
-            temp = float(self.cleaned_data['temp'])
-            database.load('solvation')
-            db = database.get_solvation_database('', '')
-            solvent_temp = self.cleaned_data['solvent_temp']
-            Tc = db.get_solvent_data(solvent_temp).get_solvent_critical_temperature()
             temp = float(temp)
-            if temp < 280 or temp >= Tc:
-                import traceback
-                traceback.print_exc()
-                raise forms.ValidationError('Temperature is out of the valid range.')
         except:
-            import traceback
-            traceback.print_exc()
-            raise forms.ValidationError('Invalid temperature.')
+            raise forms.ValidationError('Invalid Temperature')
+        database.load('solvation')
+        db = database.get_solvation_database('', '')
+        solvent_temp = self.cleaned_data['solvent_temp']
+        if solvent_temp == '':
+            pass
+        else:
+            Tc = db.get_solvent_data(solvent_temp).get_solvent_critical_temperature()
+            if temp < 280 or temp >= Tc:
+                raise forms.ValidationError('Temperature is out of the valid range')
```

Comment:
Do you think we can use `if not solvent_temp:` to replace `if: pass else:...`

Reply:
bfb4cd7